File: app/src/main/java/com/samiraa_raghadm_sawsana/meditrack/data/Schedule.java
package com.samiraa_raghadm_sawsana.meditrack.data;

public class Schedule {

    private int id;
    private int medicationId;
    private String intakeTime;
    private String daysOfWeek;

    public Schedule() {
    }

    public Schedule(int id, int medicationId, String intakeTime, String daysOfWeek) {
        this.id = id;
        this.medicationId = medicationId;
        this.intakeTime = intakeTime;
        this.daysOfWeek = daysOfWeek;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getMedicationId() {
        return medicationId;
    }

    public void setMedicationId(int medicationId) {
        this.medicationId = medicationId;
    }

    public String getIntakeTime() {
        return intakeTime;
    }

    public void setIntakeTime(String intakeTime) {
        this.intakeTime = intakeTime;
    }

    public String getDaysOfWeek() {
        return daysOfWeek;
    }

    public void setDaysOfWeek(String daysOfWeek) {
        this.daysOfWeek = daysOfWeek;
    }
}

File: app/src/main/java/com/meditrack/app/models/Schedule.kt
package com.meditrack.app.models

/**
 * מודל לוח זמנים - מתי לקחת תרופה מסוימת
 */
data class Schedule(
    val id: Long = 0,
    val medicationId: Long,   // FK → Medication
    val hour: Int,            // שעה (0-23)
    val minute: Int,          // דקות (0-59)
    val daysOfWeek: String    // "1,2,3,4,5,6,7" = ימים (1=ראשון ... 7=שבת)
)
